projects/1.19/assets/hex-keys/hexkeys/lang/zh_cn.json

```
@@ -0,0 +1,28 @@
+{
+  "block.hexkeys.mindscape_barrier": "意识屏障",
+  "hexcasting.spell.hexkeys:library": "图书馆钥匙",
+  "hexcasting.spell.hexkeys:visit_library": "访问者钥匙",
+  "hexcasting.spell.hexkeys:create_library_key": "墨菲斯之精思",
+  "hexcasting.spell.hexkeys:kleins_chest": "克莱因箱",
+  "hexkeys.entry.klein_storage": "克莱因存储",
+  "hexkeys.entry.visit_mindscape": "进入意识界",
+  "hexkeys.entry.mindscape": "意识界图书馆",
+  "container.kleins_chest": "克莱因箱",
+  "hexkeys.page.kleins.1": "我似乎找到了将物质完全转换为媒质的方式，还能保留其原本的属性。但可惜，我也只能在意识中存储有限个物品，意识里只有那么多空间。我称这种独特的存储方式为“克莱因箱”，它是我意识中的空间，且腾出的空位和要存入的事物似乎完全对应。",

```

最后一句的意思感觉是说，Klein瓶子装多少漏多少（因为没有内部），「我」的意识也是如此（
```suggestion
  "hexkeys.page.kleins.1": "我似乎找到了将物质完全转换为媒质的方式，还能保留其原本的属性。但可惜，我也只能在意识中存储有限个物品，毕竟意识里只有那么多空间。我称这种独特的存储方式为“克莱因箱”，因为它是我意识中的空间，并且装多少漏多少。",
```

---

当时看这个描述其实挺奇怪的，因为Klein箱基本就是54槽的末影箱，确实是不会漏
Klein瓶的确，可能算是empty的某种双关梗？

前面的部分已改

---

那大概不是游戏机制而是现实意义上的吐槽（